lib/database.cpp

```diff
@@ -405,7 +405,7 @@ void Database::setDevicesReceivedKey(const QString& roomId, const QVector<std::t
         query.bindValue(":userId", user);
         query.bindValue(":deviceId", device);
         query.bindValue(":identityKey", curveKey);
-        query.bindValue(":sessionId", sessionId);
+        query.bindValue(":sessionId", sessionId.toLatin1());
```

Comment:
...and here too: `QByteArray` is already passed to `setDevicesReceivedKey()`, using `QString` here is not right (any more, since the caller, `Connection::sendSessionKeyToDevices()`, switched to `QByteArray`). And similar to the previous case, it would probably be best if `sendSessionKeyToDevices()` just accepted `QOlmOutboundGroupSession` as a parameter instead of a bunch of its attributes, and passed over that session further to `Database`.